site/sfguides/src/community_detection_using_relationalai/community_detection_using_relationalai.md

```diff
@@ -26,7 +26,7 @@ Users can build a knowledge graph using Python, and materialize it on top of the
 - A [Snowflake](https://signup.snowflake.com/) Account
 - Basic knowledge of using a Snowflake SQL Worksheet
 - Snowflake privileges on your user to [Install a Native Application](https://other-docs.snowflake.com/en/native-apps/consumer-installing#set-up-required-privileges)
-- A Database named `FROSTBYTE_TASTY_BYTES` with the "TastyBytes" demo data set, which can be installed using these [instructions](https://quickstarts.snowflake.com/guide/tasty_bytes_introduction/index.html#1)
+- A Database named `FROSTBYTE_TASTY_BYTES` with the "TastyBytes" demo data set. To get access to this data, please send an email request to `sales@relational.ai`, with your SF account identifier for us to share the data
 - The [RAI Community Detection Jupyter notebook](https://github.com/RelationalAI/rai-samples/blob/main/samples/tastybytes-communities/community_detection_RelationalAI_V1.ipynb) used in this quickstart
```

Comment:
Needs to point to 
https://github.com/RelationalAI/rai-samples/blob/main/samples/tastybytes-communities/community_detection_RelationalAI_0.3.ipynb

Reply:
done